app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -167,6 +167,22 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
         vm.cloudVolumeModel.aws_volume_type !== 'standard');
   };
 
+  vm.awsBaseSnapshotChanged = function(baseSnapshotId) {
+    vm.encryptionDisabled = false;
+    $timeout(function() {
+      if (baseSnapshotId) {
+        for (var i = 0; i < vm.baseSnapshotChoices.length; i++) {
+          if (vm.baseSnapshotChoices[i].ems_ref === baseSnapshotId) {
+            vm.cloudVolumeModel.aws_encryption = vm.baseSnapshotChoices[i].encrypted;
```

Comment:
@AparnaKarve Any reason for suggesting the use of `angular.isDefined`? Shouldn't this do the trick?

```
vm.baseSnapshotChoices[i].encrypted === true
```
Example: https://jsfiddle.net/aj1rkyev/1/

@sasoc Please, take a look at the change that you have made in your [last push](https://github.com/ManageIQ/manageiq-ui-classic/pull/1651/files#diff-638665f4b337f59cc291a102efee0ce6R176).